src/main/java/ai/shreds/domain/DomainSupplierEntity.java

```diff
@@ -0,0 +1,37 @@
+package ai.shreds.domain;
+
+import lombok.Data;
+import org.hibernate.annotations.CreationTimestamp;
+import org.hibernate.annotations.UpdateTimestamp;
+import javax.persistence.*;
+import java.time.LocalDateTime;
+import javax.validation.constraints.Size;
+
+@Data
+@Entity
+@Table(name = "Supplier")
+public class DomainSupplierEntity {
```

Comment:
add to the DomainSupplierEntity:
1- due date for the supplier payement
2- company name

propage the change to all other dto classes and classes that use/map the entity class (SharedSupplierDTO, AdapterSupplierMapper, ...)